components/question/Details/index.jsx

```diff
@@ -0,0 +1,31 @@
+import PropTypes from 'prop-types';
+import { useEffect } from 'react';
+import { useRouter } from 'next/router';
+
+import useUser from 'hooks/useUser';
+import useSurvey from 'hooks/useSurvey';
+
+const QuestionDetails = ({ setBackgroundImagePath }) => {
+  const router = useRouter();
+  const { id } = router.query;
+  const { user } = useUser();
+  const survey = useSurvey(user, id);
+
+  useEffect(() => {
+    if (survey) {
+      setBackgroundImagePath(survey.data.attributes.coverImageUrl);
+    }
+  }, [survey, setBackgroundImagePath]);
+
+  if (!survey) {
+    return null;
+  }
+
+  return <div>Hello World</div>;
```

Comment:
```suggestion
  return <div>TODO: Question rendering goes here</div>;
```

While I have no problem with the "hello world", it can be interesting to utilize this space for providing indications of the intent of what would be rendered later in here :)

Reply:
done in 0a12e6f2768d6cf6c93db4a407b806b7fc73a764